frontend/src/metabase/dashboard/components/ClickBehaviorSidebar/ActionOptions/ActionOptions.styled.tsx

```diff
@@ -32,3 +32,10 @@ export const ActionDescription = styled.span<{ isSelected?: boolean }>`
 export const ClickMappingsContainer = styled.div`
   margin-top: 1rem;
 `;
+
+export const ActionPickerWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  max-height: calc(100vh - 30rem);
```

Comment:
this ain't great, but I suspect this sidebar is gonna see some significant revisions soon, and I didn't want to optimize this too much just to throw it away in a couple days.